src/Evolution/Initialization/InterfaceNewtonianEuler.hpp

```diff
@@ -0,0 +1,103 @@
+// Distributed under the MIT License.
```

Comment:
Two points on this:

- Why is this necessary? It should be clearly stated in the documentation why the current generic one isn't used.
- This should be in the NE directory since it's system specific.